cfy_manager/components/restservice/restservice.py

```diff
@@ -287,6 +290,26 @@ def _configure():
     _start_restservice()
 
 
+def _remove_files():
+    """
+    Remove all files related to the REST service and uninstall the RPM,
+    but don't remove the spec files which are handled by the installation RPM
+    """
+    tmp_dir = mkdtemp()
+    # Keep the spec files in a temp location
+    common.move(join(constants.MANAGER_RESOURCES_HOME, 'spec'), tmp_dir)
+
+    # Removing the RPM before recreating /opt/manager/resources, because
+    # yum remove will delete this folder
+    yum_remove('cloudify-rest-service')
```

Comment:
@mcouthon this had had the `remove_files([HOME_DIR, LOG_DIR, RESTSERVICE_RESOURCES])` line removed (in the merge commit)
i think that is correct because those should all be removed in the rpm